docs/dynamic-links/usage/index.md

```diff
@@ -7,7 +7,7 @@ previous: /database/presence-detection
 ---
 
 > [!WARNING]
-> Deprecated: Firebase Dynamic Links is deprecated and should not be adopted in projects that don't already use it. The service will shut down on August 25, 2025. See the [Dynamic Links Deprecation FAQ](https://firebase.google.com/support/dynamic-links-faq) for more information.
+> Deprecated: Firebase Dynamic Links is deprecated and should not be adopted in projects that don't already use it. Functionality for the primary use cases of store/web routing and deferred/regular deep-linking will shut down on August 25, 2025. Firebase Authentication currently uses, and will continue to use, Firebase Dynamic Links to customize Authentication links (passwordless sign in, password reset, etc.) beyond the August 25, 2025 deadline. See the [Dynamic Links Deprecation FAQ](https://firebase.google.com/support/dynamic-links-faq) for more information.
```

Comment:
The text upstream is slightly different than what was proposed here and does not make the same guarantee - that is they do not commit to using Dynamic Links for this functionality, they just commit to a vague "update" to make sure it keeps working. I've aligned the text here with the current text upstream

```suggestion
> Deprecated: Firebase Dynamic Links is deprecated and should not be adopted in projects that don't already use it. Functionality for the primary use cases of store/web routing and deferred/regular deep-linking will shut down on August 25, 2025. Firebase Authentication currently uses Firebase Dynamic Links to customize Authentication links, but the Firebase team will provide an update to ensure that this functionality continues working after the Firebase Dynamic Links service is shut down. See the [Dynamic Links Deprecation FAQ](https://firebase.google.com/support/dynamic-links-faq) for more information.
```